azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -193,18 +196,22 @@ public AzkabanWebServer() throws Exception {
    * Constructor
    */
   public AzkabanWebServer(Server server, Props props) throws Exception {
```

Comment:
The constructor is used externally. It is complicated and that's why I didn't refactor this part as part of this change.